solutions/usaco-966.mdx

```diff
@@ -47,4 +47,48 @@ ans += first8[N % 8]
 print(ans)
 ```
 </PySection>
+<JavaSection>
+  
+```java
+
+import java.util.*;
+import java.io.*;
+
+
+class usaco-966
```

Comment:
```suggestion
public class MooBuzz
```
With "Problem name" I meant `public class MooBuzz`, Java won't compile with slash in class name.
Ah and if class isn't public it won't work anyway